File: Appendix M - Source Code/Tier 2 (Java)/java_rest/src/main/java/utilities/objects/Logedin.java
package utilities.objects;

/**
 * Class for holding the boolean objects, needed for proper functioning of Json Serialization/Deserialization
 * @author Group6
 * @version 0.1.0
 */
public class Logedin
{
    private boolean logedin;

    /**
     * No argument constructor used for Json Serialization/Deserialization
     */
    public Logedin()
    {
        super();
    }

    /**
     * Base constructor
     * @param logedin
     */
    public Logedin(boolean logedin)
    {
        this.logedin = logedin;
    }

    public boolean isLogedin()
    {
        return logedin;
    }

    public void setLogedin(boolean logedin)
    {
        this.logedin = logedin;
    }
}

File: Appendix M - Source Code/Tier 1 (C#)/TheHappyPig/TheHappyPig/Utility/Logedin.cs
using System;

namespace TheHappyPig.Model.Utility
{
    public class Logedin
    {
        public bool logedin { get; set; }

        public Logedin() : base()
        {

        }

        public Logedin(bool logedin)
        {
            this.logedin = logedin;
        }
    }
}
